Tasks/AzurePowerShellV2/TryMakingModuleAvailable.ps1

```diff
@@ -0,0 +1,33 @@
+[CmdletBinding()]
+param (
+    [string]
+    $targetVersion
+)
+
+. "$PSScriptRoot\Utility.ps1"
+$moduleContainerPath = Get-SavedModuleContainerPath
+
+if (-not(Test-Path -Path $moduleContainerPath)) {
+    return;
+}
+
+if (!$targetVersion) {
+    return;
+}
+
+# value for classic
+@($false, $true).ForEach({
```

Comment:
can we have static const list and we iterate over it for safe side, as we are sure we are not going to update any of classic modules we dont need this generic logic

Reply:
Can you please elaborate as I did not get this comment?

Reply:
As explained this is not generic logic and just a way to pass classic value to run the code twice. resolving.